Source/SLLibMacros.h

```diff
@@ -0,0 +1,13 @@
+//
+//  SLLibMacros.h
+//  LoggingDemo
+//
+//  Created by 利辺羅 on 2014/12/10.
+//  Copyright (c) 2014年 SLF4C. All rights reserved.
+//
+
+#import "SLMacros.h"
+
+#undef  SLMarkerName
+#define SLMarkerName @"Other"
```

Comment:
Let me get this straight: the idea is the default logging will use `'Other'` unless the SLMarkerName is redefined. This sounds right to me